setup.py

```diff
@@ -36,20 +37,24 @@
 with open("README.md") as file:
     long_description = file.read()
 
-if check_for_openmp():
-    omp_args = ["-fopenmp"]
-else:
-    omp_args = []
+CPP14_CONFIG = defaultdict(
+    lambda: ["-std=c++14"], {"unix": ["-std=c++14"], "msvc": ["/std:c++14"]}
+)
+CPP03_CONFIG = defaultdict(
+    lambda: ["-std=c++03"], {"unix": ["-std=c++03"], "msvc": ["/std:c++03"]}
+)
+
+_COMPILER = get_default_compiler()
+
+omp_args, _ = check_for_openmp()
```

Comment:
If we are distinguishing between compile and link flags inside `check_for_openmp()`, shouldn't we do the same thing during cythonization, i.e. use two separate aliases? Otherwise, why return two variables?